tools/SetupFlow/DevHome.SetupFlow/ViewModels/ReviewViewModel.cs

```diff
@@ -215,4 +240,37 @@ private void ReportGenerateConfiguration()
         var nonInstalledPackagesCount = installTasks.Count() - installedPackagesCount;
         TelemetryFactory.Get<ITelemetry>().Log("Review_GenerateConfigurationForInstallPackages", LogLevel.Critical, new ReviewGenerateConfigurationForInstallEvent(installedPackagesCount, nonInstalledPackagesCount));
     }
+
+    /// <summary>
+    /// Receives the adaptive card message and checks if its for the review view model.
+    /// </summary>
+    public async void Receive(NewAdaptiveCardAvailableMessage message)
+    {
+        InfoBarErrorMessage = null;

```

Comment:
Should this be reset even if the message isn't from a ReviewViewModel?

Reply:
fair, will move it down